src/generate/convert/definition.rs

```diff
@@ -42,7 +42,12 @@ pub fn convert_def(ast: &ASTTy, imp: &mut Imports, state: &State, ctx: &Context)
                         match (ty, expression) {
                             (Some(ty), _) => Some(Box::from(convert_node(ty, imp, &state, ctx)?)),
                             (_, Some(expr)) if state.annotate => {
-                                expr.clone().ty.map(|name| name.to_py(imp)).map(Box::from)
+                                let ty = expr.clone().ty.map(|name| name.to_py(imp)).map(Box::from);
+                                if let Some(core) = ty {
```

Comment:
Clippy warning aside, this logic ideally shouldn't be necessary.
Why do we even get `Core::Empty` in the first place?